meson.build

```diff
@@ -32,7 +32,8 @@ sources = [
     'src/Rendering.cpp',
     'src/SpriteRendering.cpp',
     'src/Windowing.cpp',
-    'src/Main.cpp'
+    'src/Main.cpp',
+    'src/LevelManager.cpp'
```

Comment:
Everything was sorted alphabetically except for the Main.cpp. It would be nice to follow that convention.